package.json

```diff
@@ -69,7 +69,7 @@
         "@wdio/spec-reporter": "6.11.0",
         "@wdio/sync": "6.10.4",
         "babel-jest": "27.3.1",
-        "chromedriver": "96.0.0",
+        "chromedriver": "^103.0.0",
```

Comment:
@jasonschroeder-sfdc - you have a Control char here '^'.  Will this work fine?

Reply:
Yep! https://rossta.net/blog/npm-version-constraints-for-rubyists.html